programmers/신규-아이디-추천.js

```diff
@@ -0,0 +1,16 @@
+function solution(id) {
+    const lowerId = id.toLowerCase()
+    let newId = lowerId.replace(/[^-_.a-z0-9]/g,'').replace(/\.{2,}/gm,'.').replace(/^\.|\.$/g,'')
```

Comment:
``` const lowerId = id.toLowerCase() ```를 굳이 선언하지 않고 아래와 같이 바로 하는것이 나음

줄바꿈을 해야 다른사람들이 보기가 편함
```suggestion
    
    let newId = id.toLowerCase()
    .replace(/[^-_.a-z0-9]/g,'')
    .replace(/\.{2,}/gm,'.')
    .replace(/^\.|\.$/g,'')
```